lib/util.js

```diff
@@ -1,3 +1,5 @@
+exports.KEY_SEPARATOR = ',016c2cec,'
```

Comment:
There must be better separators than a commit hash? Like some single character that is not allowed in a table key?

Reply:
I try to find this kind of character, but a string table key accepts every character. So I use combined characters to make sure that it is unique.

Reply:
It's a breaking change because links from previous versions will not work with this version and vice-versa. So if we are breaking things, we might as well just refactor it in a more sane way. For example encode the key somehow. I don't like using random sha as a string separator.